hc_core/src/nucleus/fncall.rs

```diff
@@ -0,0 +1,19 @@
+/// Represents a function call, with the function name and its parameters
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Params {}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Call {
```

Comment:
Hm, I have basically the same thing in my ribosome branch.. we should take care not to duplicate work.

Reply:
@lucksus yeah i saw that in your screen share, a little duplication of work is ok while everything is so new, but we should agree on the details :)